common.blocks/icon/_view/icon_view_markdown.css

```diff
@@ -0,0 +1,3 @@
+.icon_view_markdown {
+    background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="208" height="128" viewBox="0 0 208 128"><rect width="198" height="118" x="5" y="5" ry="10" stroke="#000" stroke-width="10" fill="none"/><path d="M30 98V30h20l20 25 20-25h20v68H90V59L70 84 50 59v39zm125 0l-30-33h20V30h20v35h20z"/></svg>');
```

Comment:
it's better to keep icons as separate files. build system will inline them automatically